contracts/predeploys.go

```diff
@@ -1,25 +1,35 @@
 package contracts
 
 import (
-	"github.com/ethereum-optimism/optimism/op-bindings/predeploys"
+	opcontracts "github.com/ethereum-optimism/optimism/op-bindings/predeploys"
 	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/polymerdao/monomer/bindings/generated"
 )
 
 type Predeploy struct {
-	Address          common.Address
-	DeployedBytecode []byte
+	address          common.Address
+	deployedBytecode []byte
 }
 
 var L2ApplicationStateRootProviderAddr = common.HexToAddress("0x4300000000000000000000000000000000000001")
 
-var Predeploys = []*Predeploy{
+var predeploys = []*Predeploy{
 	{
-		Address:          L2ApplicationStateRootProviderAddr,
-		DeployedBytecode: common.FromHex(bindings.L2ApplicationStateRootProviderMetaData.Bin),
+		address:          L2ApplicationStateRootProviderAddr,
+		deployedBytecode: common.FromHex(bindings.L2ApplicationStateRootProviderMetaData.Bin),
 	},
 	{
-		Address:          predeploys.L2ToL1MessagePasserAddr,
-		DeployedBytecode: common.FromHex(bindings.L2ToL1MessagePasserMetaData.Bin),
+		address:          opcontracts.L2ToL1MessagePasserAddr,
+		deployedBytecode: common.FromHex(bindings.L2ToL1MessagePasserMetaData.Bin),
 	},
 }
+
+func PredeployContracts(ethState *state.StateDB) *state.StateDB {
```

Comment:
nit: go style avoid stutters `contracts.Predeploy` vs `contracts.PredeployContracts`
```suggestion
func Predeploy(ethState *state.StateDB) *state.StateDB {
```